doc/user/content/cli/_index.md

```diff
@@ -269,10 +269,11 @@ logged.
 
 Information reported to Materialize:
 
-* Cluster ID
+* Cluster and Session IDs
```

Comment:
Are these explained somewhere? This is overloading of the term "session", right? Since it is unrelated to SQL sessions.

Reply:
They are not, that's a good point, I'll expand on them.